github/users.go

```diff
@@ -166,3 +166,59 @@ func (s *UsersService) ListAll(opt *UserListOptions) ([]User, *Response, error)
 
 	return *users, resp, err
 }
+
+// ListInvitations lists all currently-open repository invitations for the
+// authenticated user.
+//
+// GitHub API docs: https://developer.github.com/v3/repos/invitations/#list-invitations-for-a-repository
+func (s *UsersService) ListInvitations() ([]RepositoryInvitation, *Response, error) {
```

Comment:
Given #375, wouldn't you want to return `[]*RepositoryInvitation` here rather than `[]RepositoryInvitation`?